frontend/src/features/data_entry/components/DataEntrySection.tsx

```diff
@@ -59,40 +59,34 @@ export function DataEntrySection({ sectionId }: { sectionId: FormSectionId }) {
     }
   }
 
-  const [missingTotalError, setMissingTotalError] = React.useState(false);
-
   // Memoize getCodes() results to prevent unnecessary focus triggers in Feedback
-  const memoizedErrorCodes = React.useMemo(() => formSection.errors.getCodes(), [formSection.errors]);
+  const memoizedErrorCodes = React.useMemo(() => {
+    const errorCodes = formSection.errors.getCodes();
+    const missingTotalErrorIndex = errorCodes.indexOf("F402", 0);
+    if (missingTotalErrorIndex > -1) {
+      errorCodes.splice(missingTotalErrorIndex, 1);
+    }
+    return errorCodes;
+  }, [formSection.errors]);
```

Comment:
I think this can be simplified to (untested):
```suggestion
  const memoizedErrorCodes = React.useMemo(
    () => formSection.errors.getCodes().filter((code) => code !== "F402"),
    [formSection.errors],
  );
```

Reply:
Yes! Changed in 5189b7a9a84250790e27f5605731d1a1783bd888